src/styles/GlobalStyles.ts

```diff
@@ -96,7 +96,7 @@ video {
   font: inherit;
   vertical-align: baseline;
   font-family: "SFProText-Regular", "Helvetica Neue", Helvetica, Arial;
-  line-height: 4rem;
+  line-height: 1.47;
```

Comment:
Hmm, not sure we want this specific line-height 🙏

Reply:
I beg to differ. 

This is with `4rem`
![4rem](https://user-images.githubusercontent.com/21690115/85940703-906ff500-b950-11ea-931d-68a1e51e3514.gif)

This is with `1.47`
![1.47](https://user-images.githubusercontent.com/21690115/85940711-a54c8880-b950-11ea-87e6-a4fc7bb3d02a.gif)

Notice the difference?

Reply:
That's fine, I mean, we should have it based on `rem`. So whatever 1.47 is, we should convert it to `rem` if possible.